wandb/sdk/wandb_init.py

```diff
@@ -418,8 +433,19 @@ def init(self) -> Union[Run, RunDisabled, None]:  # noqa: C901
                         )
                     )
         elif isinstance(wandb.run, Run):
-            logger.info("wandb.init() called when a run is still active")
-            return wandb.run
+            if wandb.run._init_pid == os.getpid():
+                logger.info("wandb.init() called when a run is still active")
+                return wandb.run
+            elif s._concurrency and not s._attach_id:
+                logger.error(
+                    "wandb.init() called when a run is still active. Unsafe mp usage."
```

Comment:
Likely add a doc link here.

Reply:
yes, i have a new module that consolidates messages related to concurrency... coming soon